src/pykeen/sampling/negative_sampler.py

```diff
@@ -25,14 +25,20 @@ def __init__(
         self,
         triples_factory: TriplesFactory,
         num_negs_per_pos: Optional[int] = None,
+        filtered: bool = False,
     ) -> None:
         """Initialize the negative sampler with the given entities.
 
         :param triples_factory: The factory holding the triples to sample from
         :param num_negs_per_pos: Number of negative samples to make per positive triple. Defaults to 1.
+        :param filtered: Whether proposed corrupted triples that are in the training data should be filtered.
+            Defaults to False.
```

Comment:
can you provide some commentary here on why the default is false and why this isn't usually a big deal?

Reply:
It looks like you've already written it up in the docstring of a private function below, but it would be good for users to see it in the docs too

Reply:
Resolved in 97a76fc9